Project/GNU/dvanalyzer.spec

```diff
@@ -67,7 +67,7 @@ particularly useful in documenting source material of edited DV content.
 Summary:	Supplies technical and tag information about a video or audio file (GUI)
 Group:		Productivity/Multimedia/Other
 
-%if 0%{?fedora_version}
+%if 0%{?fedora_version} || 0%{?centos} > 6
```

Comment:
I prefer `>= ` in order to be more coherent (e.g. `%if 0%{?rhel} >= 7` below, despite the fact I missed `0%{?mageia} > 5` above :( )

Reply:
Fixed